app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,15 @@
 class PostsController < ApplicationController
```

Comment:
- [ ] Kindly solve this error that is preventing you migration to run. this is happening because the create like migration is created before a table that it relates to(posts). kindly fix this 

![image](https://github.com/LarryIVC/larrys-blog/assets/65068771/8f34170e-5ab4-4d66-b743-0e11ad052a53)

Reply:
Hi @andersonlebon, do you happen to have a hint to correct this error, I don't know why it happened.

Reply:
Okay

Reply:
Hello @LarryIVC, yeah I have told you this is happening because the create like micration that has the relationship with the post is coming before the create post migration. So you refere post that hasn't yet been created.